content/tutorials/drift-detection-and-remediation/index.md

```diff
@@ -0,0 +1,224 @@
+   ip      : "18.236.205.169"
+   url     : "http://ec2-18-236-205-169.us-west-2.compute.amazonaws.com"
+
+Resources:
+   + 8 created
+```
+
+Now you can view your new stack in Pulumi Cloud by following the **View in Browser** link. Click on the **Drift** tab and you will see that Drift is unavailable as your stack's deployments are not yet configured. Click on the **Configure deployments setting** to see your Deployments source control settings.
+
+To use Drift Detection with Deployments you will first need to create a new GitHub repository.
+
+## Create a new repository on GitHub
+
+1. Go to [GitHub](https://github.com/new) and create a new repository manually.
+2. Copy the repository URL provided by GitHub.
+3. Replace the origin with your new GitHub repo using the following commands:
+
+```bash
+git init
+git add .
+git commit -m "Initial commit for my Pulumi Drift tutorial"
+git branch -M main
+git remote add origin https://github.com/your-username/learn-pulumi-drift-detection.git
+git push -u origin main
+```
+
+## Configure Drift Detection
+
+Now you will return to your browser to set your deployment configuration source control settings. Navigate to your stack `Settings` and `Deploy` tab. Next, configure the required Deployment settings, including your source control.
+
+1. Add your GitHub organization/repository and the branch you created.
+
+2. Add your Secret environment variables, including `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.
+
+{{% notes type="info" %}}
+Secret environment variables, such as `AWS_SECRET_ACCESS_KEY`, are encrypted end-to-end with Pulumi and can be set on each stack. However, by creating an [environment](/docs/pulumi-cloud/esc) with Pulumi ESC, you can centralize secrets and set up OIDC for secure authentication. This allows you to manage and share sensitive configuration data across multiple stacks efficiently.
+{{% /notes %}}
+
+Click **Save deployment configuration**.
+
+<video autoplay loop muted playsinline>
+    <source src="pulumi-configure-drift.mp4" type="video/mp4">
+    Your browser does not support the video tag.
+</video>
+
+Once you have configured Pulumi Deployments, open the Actions menu and choose `Detect Drift`, then click `Deploy` to validate your stack, program and deployments are all running before we introduce drift.
+
+![Screenshot of the Pulumi Cloud console to run a Drift Detection](./pulumi-detect-drift-action.png)
+
+After the drift detection is complete, you will see a new Drift run on the `Drift` tab timeline. Since this is the initial check and no deviations have been introduced, you will not receive any notifications indicating that drift has been detected, nor will you see any resources with changes.
+
+## Introduce drift
+
+To see Drift Detection in action, you will now introduce drift by manually making changes to your newly created EC2 resource, in this case by adding a new tag.
+
+### Add a tag to EC2 using AWS CLI
```

Comment:
Might be a little verbose, but sounds more accurate imo as EC2 is the general name for the aws service which encompasses a bunch of other resources that fall under it, an EC2 instance being one of them.

```suggestion
### Add a tag to the EC2 instance using AWS CLI
```

Reply:
done, thx